common/rc

```diff
@@ -4538,7 +4538,7 @@ init_rc()
 
 	# Sanity check that TEST partition is not mounted at another mount point
 	# or as another fs type
-	_check_mounted_on TEST_DEV $TEST_DEV TEST_DIR $TEST_DIR $FSTYP || exit 1
+	_check_mounted_on TEST_DEV $TEST_DEV TEST_DIR $TEST_DIR $FSTYP$FUSE_SUBTYP || exit 1
 	if [ -n "$SCRATCH_DEV" ]; then
```

Comment:
Where is FUSE_SUBTYPE defined? In local.config? Also needs to go to the README then?

Reply:
its already defined in local.config. I am not getting which README file you are talking about other than above one

Reply:
Ok, I had missed that